shuup/core/models/_orders.py

```diff
@@ -797,7 +798,7 @@ def get_product_summary(self):
             products[product_id]['unshipped'] -= quantity
 
         refunded_prods = self.lines.refunds().filter(
-            type=OrderLineType.QUANTITY_REFUND,
+            type=OrderLineType.REFUND,
```

Comment:
Again, it seems that only products are refunded? I understand that refunding in a sense of "returning items to stock" only considers products, but how about rest of the lines?